dynamic_models/models.py

```diff
@@ -11,25 +11,36 @@
 
 
 class ModelSchema(models.Model):
-    name = models.CharField(max_length=32, unique=True)
+    name = models.CharField(max_length=250, unique=True)
     db_name = models.CharField(max_length=32, default=DEFAULT_DB_ALIAS)
+    managed = models.BooleanField(default=True)
+    db_table_name = models.CharField(null=True, max_length=250)
```

Comment:
**question**: can you explain the reasoning for 250 max length? In postgres the name of a table is capped at 63 chars.

Similarly, are you aware of any hard cap for the model name length? If not we can probably use `TextField` instead

Reply:
hm, I see. Not sure if we should set a  max size here unless there's a meaningful value. 250 seems safe enough if we have to choose one.